src/openms/source/ANALYSIS/OPENSWATH/SwathMapMassCorrection.cpp

```diff
@@ -79,6 +79,36 @@ namespace OpenMS
     return used_maps;
   }
 
+  // Checks SwathMaps for PASEF data in which data can overlap across IM #TODO add documentation
+  std::vector<OpenSwath::SwathMap> findSwathMapsPasef(const OpenMS::MRMFeatureFinderScoring::MRMTransitionGroupType& transition_group,
+                                                 const std::vector< OpenSwath::SwathMap > & swath_maps)
+  {
+    // Although theoretically there can be more than one map, for this case, just use the "best" map, best map is defined as the one in which the IM is closest to the center of the window
+    std::vector<OpenSwath::SwathMap> used_maps;
```

Comment:
why does this need to be a vector? looks like the return value has at most size 1

Reply:
the return value is expected to be a vector here